lang/tags/functions_common.py

```diff
@@ -25,15 +25,15 @@ def code_toggle_functions():
         if gui_functions.showing:
             function_list = []
             gui_functions.hide()
-            ctx.tags.discard("user.code_functions_gui_showing")
+            ctx.tags.discard("user.code_functions_common_gui_active")
```

Comment:
This method doesn't exist, because `ctx.tags` is a frozenset (immutable) - the right approach is to set `ctx.tags` to an updated set, `ctx.tags = ...`. Since the code is already broken, no objection to merging, but would be nice to fix this at some point.